lib/rouge/lexers/systemd.rb

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*- #
+# frozen_string_literal: true
+
+module Rouge
+  module Lexers
+    class SystemD < RegexLexer
+      tag 'systemd'
+      aliases 'unit-file'
+      filenames '*.service', '*.socket', '*.device', '*.mount', '*.automount', '*.swap', '*.target', '*.path', '*.timer', '*.slice', '*.scope'
```

Comment:
Some of these filenames seem like names that may be used by other programs. Would it be insufficient to just cut this back to `*.service`?

Reply:
These extensions are for all the unit configuration types supported (see "Description" section in https://www.freedesktop.org/software/systemd/man/systemd.unit.html).
I guess some of them may be used by other programs but does it matter?

Reply:
@jljouannic Rouge is the syntax highlighting library used by GitLab and I believe that the way they have it configured, these file globs would be picked up and used to syntax highlight files in GitLab repositories. Since it's easier to add this kind of thing later than remove it, my suggestion is that we limit the list of file globs for now to `*.service` and then look to add the others based on user feedback.